doc/develop/test/twister.rst

```diff
@@ -1278,6 +1278,22 @@ using an external J-Link probe.  The ``probe_id`` keyword overrides the
       runner: jlink
       serial: null
 
+Using One Board For Variants
+++++++++++++++++++++++++++++
+
+  If you want to run tests for different variants on the same board
+  without re-configuring the hardware map file for each variant, you can
+  extend the ``platform`` attribute with names separated by commas.
+
+.. code-block:: yaml
+
+    - connected: true
+      id: '001234567890'
+      platform: nrf5340dk/nrf5340/cpuapp, nrf5340dk/nrf5340/cpuapp/ns
```

Comment:
can this instead be a list with `type: any` in the schema so it does support one single entry and a list if variants if needed?

Reply:
I considered using:
```
type: seq
sequence:
    - type str
```
but that would break backward compatibility, so I didn't apply it.

Good idea with `type: any`, I will try to update it.
I was looking into testsuite-schema and there are a few fields that can be either a string or a list. There is also a logic in `config_parser.py` to properly cast those fields.  I need to look into it more closely.
Thanks for the advice.

Reply:
Updated with  `type: any`. Now user can use a list of platforms:
in line, separated by spaces:
```
platform: nrf5340dk/nrf5340/cpuapp nrf5340dk/nrf5340/cpuapp/ns
```
or as a list:
```
platform:
- nrf5340dk/nrf5340/cpuapp
- nrf5340dk/nrf5340/cpuapp/ns
```